packages/components/src/components/icon-with-message/icon-with-message.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classNames from 'classnames';
+import { isDesktop } from '@deriv/shared';
+import Text from '../text/text';
+import Div100vhContainer from '../div100vh-container';
+
+type TIconWithMessage = {
+    className?: string;
+    full_width?: boolean;
+    icon: React.ReactElement;
+    message: React.ReactNode;
+    text?: string | React.ReactElement;
+};
+
+const IconWithMessage = ({
```

Comment:
there might be confusions as we have component in Accounts with the same name IconWithMessage.
it is necessary to keep both of them?

Reply:
Have to look into this 👍 cause it was previously created for DIEL project but now in master there's something similar 🙏

Reply:
@yauheni-deriv After checking the one's in account and in components 2 are different thing. The display style also different. Will change 1 of the component's name instead 🙏